.php-cs-fixer.dist.php

```diff
@@ -1,8 +1,74 @@
 <?php
 
-$config = \TYPO3\CodingStandards\CsFixerConfig::create();
-$config
-    ->getFinder()->in(__DIR__)
-;
+declare(strict_types=1);
 
-return $config;
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+return (new Config())
+    ->setFinder(
+        (new Finder())
+            ->in(__DIR__)
+    )
+    ->setRiskyAllowed(true)
+    ->setRules([
```

Comment:
Where do all these extra rules come from?

Reply:
As written in the commit message, I am using the rules from Core:
https://github.com/TYPO3/typo3/blob/main/Build/php-cs-fixer/config.php

But we can stick with PER-CS@1 and small adjustments like in render-guides.

Reply:
ah ok, Ic